modules/Microsoft.EventGrid/systemTopics/eventSubscriptions/deploy.bicep

```
@@ -0,0 +1,90 @@
+@description('Required. The name of the Event Subscription.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Required. Name of the Event Grid System Topic.')
+param eventGridSystemTopicName string
```

```suggestion
param systemTopicName string
```
Wouldn't that be descriptive enough? Just figured as we shortened a lot of parameters with the current release. Note that this would need to be updated in other places too.